.github/workflows/validate-cg-manifest.sh

```diff
@@ -23,6 +24,12 @@ ignore_list=" \
   python-markupsafe \
   python-requests \
   python-zope-interface \
+  openjdk8 \
```

Comment:
Looks like the auto merge added this- we take our openjdk8 sources from a regular place now, so we can remove it from this list I believe

Reply:
Done